### PR TITLE
Appstore appmodal change

### PR DESCRIPTION
**Pull request by vinu-deriv**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Addresses (redmine-issue-number)#

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update
-   [ ] Package update

# Author Checklist:

Please check relevant items before requesting for review.

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Bahar-FS** [comment]
@vinu-deriv There is no need to replace the `imports { **** } from 'react'` to `import React from 'react';`
Is there any reason you do it?

**Comment by vinu-deriv** [comment]
> @vinu-deriv There is no need to replace the `imports { **** } from 'react'` to `import React from 'react';` Is there any reason you do it?

Yes @Bahar-FS , without using `import React from 'react';`, I was not able to resolve React is undefined error while using modal in appstore.

**Comment by Bahar-FS** [comment]
> 

I see, ok then

**Review by Bahar-FS — APPROVED** [review]

File: src/components/wallet/modal/modal-body.tsx [hunk 1]
```diff
@@ -71,6 +71,8 @@ const ModalBody = ({ children, balance, currency, dark, logo, message, message_t
                                         />
                                     </div>
                                 );
+
+                            return null;
```
[the remark this answers is not in the document]
Comment (vinu-deriv) [reply src/components/wallet/modal/modal-body.tsx:75]: Yeah, I was getting ts error something like not all code paths return a value

**Review by yashim-deriv — APPROVED** [review]

**Pull request merged**